docs/DeveloperGuide.adoc

```diff
@@ -385,6 +385,225 @@ We are using `java.util.logging` package for logging. The `LogsCenter` class is
 
 Certain properties of the application can be controlled (e.g App name, logging level) through the configuration file (default: `config.json`).
 
+=== Automatic creation of tags from person attributes
+
+==== Creation of the `Tag`
+We have made it such that the attributes `Price`, `Subject`, `Level` and `Status` of a person each have a tag with the value added to the person.
```

Comment:
Is there any particular reason for making these attributes as tags? Is there any other way to store/use them?
No need to really change anything, but would be nice if you can think through these aspects.